app/config/parameters.yml.dist

```diff
@@ -25,3 +25,7 @@ parameters:
     hypothesis_authority: some-authority
     hypothesis_client_id: some-id
     hypothesis_client_secret: ThisIsASecret
+    subject_rewrites:
+        from-subject:
```

Comment:
Not sure this format (`-` in key names) plays well if we put it into a pillar, for example.
```
subject_rewrites:
    - from: from-subject
      to: to-subject
      name: To Subject name
```
would be more robust (or how list of dictionaries are made in YAML)